src/shared/scrapers/UKR/index.js

```diff
@@ -0,0 +1,99 @@
+import * as fetch from '../../lib/fetch/index.js';
+import datetime from '../../lib/datetime/index.js';
+import maintainers from '../../lib/maintainers.js';
+import * as geography from '../../lib/geography/index.js';
+import * as parse from '../../lib/parse.js';
+
+const iso2ref = {
+  'Autonomous Republic of Crimea': 'iso2:UA-43',
+  'Cherkasy region': 'iso2:UA-71',
+  'Chernihiv region': 'iso2:UA-74',
+  'Chernivtsi region': 'iso2:UA-77',
+  'Dnipropetrovsk region': 'iso2:UA-12',
+  'Donetsk region': 'iso2:UA-14',
+  'Ivano-Frankivsk region': 'iso2:UA-26',
+  'Kharkiv region': 'iso2:UA-63',
+  'Kherson region': 'iso2:UA-65',
+
+  'Khmelnytskyi region': 'iso2:UA-68',
+  'Khmelnytsky region': 'iso2:UA-68',
+
+  'Kirovohrad region': 'iso2:UA-35',
+  'Kirovograd region': 'iso2:UA-35',
+
+  Kyiv: 'iso2:UA-30',
```

Comment:
Kyiv would never be used, would it?

Reply:
For all the duplicate values, the top one comes from the list you gave me before, the second one comes from what is in the JSON

Reply:
The list I gave is not important, only the ids on it. We just need to map to the ids. You can even do `.replace(/region/g, '')`and just make everything shorter.